lib/merchant.rb

```diff
@@ -0,0 +1,11 @@
+class Merchant
+  attr_reader :id,
+              :name
+
+  def initialize(merchant_info)
+    @id = merchant_info[:id]
```

Comment:
Great job of accepting a hash here and being able to call on the elements in a very clean and concise way.